realm/realm-library/build.gradle

```diff
@@ -19,6 +39,18 @@ android {
         project.archivesBaseName = "realm-android-library"
         consumerProguardFiles 'proguard-rules.pro'
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        externalNativeBuild {
+            cmake {
+                arguments "-DREALM_CORE_DIST_DIR:STRING=${project.coreDir.getAbsolutePath()}"
+                    abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a', 'arm64-v8a', 'mips'
```

Comment:
Yes, until further notice. It is technically a breaking change to remove it, but the plan was to remove it in 2.0 since Google has removed support for it and virtually no devices exists anymore that uses it.